lib/models.js

```diff
@@ -58,6 +58,16 @@ export class Path {
         return this.stat ? !this.stat.isDirectory() : null;
     }
 
+    isVisible() {
+      ignoredFiles = config.get('ignoredFiles');
+      if (!ignoredFiles) return true;
+      ignoredExtensions = ignoredFiles.split(',');
+      for( i in ignoredExtensions) {
```

Comment:
A [for...of loop](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of) would work nicely here.